static/js/index.js

```diff
@@ -57,7 +57,13 @@ function renderIndexDropdown(listIndices) {
     if (Cookies.get('IndexList')) {
         selectedSearchIndex = Cookies.get('IndexList');
     }else {
-        selectedSearchIndex = sortedListIndices[0].index;
-        $("#index-btn span").html(sortedListIndices[0].index);
+        let selected = sortedListIndices[0].index
+
+        if(sortedListIndices.length >= 2){
+            selected = sortedListIndices[1].index
+        }
```

Comment:
This will always choose the second index if there is a second one. Kunal said it should be the second index if the first one is named `default` (or maybe I'm just unaware of more conversation about this that you had?)

Also it would be good to have another `if` statement to check there's at least one index before you try to get the first one, unless Javascript handles it ok the way you currently have it

Reply:
Made the required changes, you may review now.